src/styles/_landingPage.scss

```diff
@@ -53,4 +53,29 @@
 
 .hide {
   display: none;
-} 
\ No newline at end of file
+} 
+
+.sun { 
+  margin-left: 201px; 
+  margin-top: 145px;
+  width:70px;
+  height:70px;
+  border-radius:50%;  
+  background: #F9F9E5;
+  opacity:0.9;      
+  box-shadow: 0px 0px 200px 70px #F9F9E5; 
+  -webkit-animation-duration: 3s;
+  -webkit-animation-name: glow;
+  -webkit-animation-direction: alternate;
+  -webkit-animation-iteration-count: infinite;
```

Comment:
is some installation needed for certain webkits to work? I checked out the branch and the animation isn't working according to the console.